doc/website/getting-started/installation.md

```diff
@@ -9,26 +9,25 @@ iceoryx_utils and iceoryx_posh are deployed as independent cmake packages. Posh
  - 64-bit hardware (e.g. x86_64 or aarch64; 32-bit hardware might work, but is not supported)
  - [cmake](https://cmake.org), 3.5 or later
  - One of the following compilers:
-   - [gcc](https://gcc.gnu.org), 7.4 or later
+   - [gcc](https://gcc.gnu.org), 7.4 or later (5.4 currently supported too)
    - [clang](https://clang.llvm.org), 9.0 or later
    - [msvc](https://visualstudio.microsoft.com/de/), part of Visual Studio 2019 or later
  - [libacl](http://download.savannah.gnu.org/releases/acl/), 2.2 or later. Only for Linux & QNX.
- - optional, [ncurses](https://invisible-island.net/ncurses/), 6.2 or later. Required by introspectiont tool.
+ - optional, [ncurses](https://invisible-island.net/ncurses/), 6.2 or later. Required by introspection tool.
 
 #### Optional, Cyclone DDS Gateway
-If you would like to use our Cyclone DDS Gateway you have to install 
-[Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) first. Furthermore
-you have to install:
+If you would like to use our Cyclone DDS Gateway you have to install [Cyclone DDS](https://github.com/eclipse-cyclonedds/cyclonedds) first.<br>
```

Comment:
In general `<br>` shouldn't be necessary. Just add a extra new line.

Reply:
Are you sure? It doesn't work for me. The markdown guide mentions to use either `<br>` or two spaces.